screens/CheckoutScreen.js

```diff
@@ -168,6 +168,18 @@ export default class CheckoutScreen extends React.Component {
     return msg;
   };
 
+  createFakeTransaction = transactionInfo => {
+    const fakeTransaction = {
```

Comment:
You can just write this as `{...transactionInfo, id: 'TRAINED', totalPrice: transactionInfo.totalSale}`

Nit: I don't know if 'TRAINED' seems like the best thing to display on Confirmation?